app/models/shift.rb

```diff
@@ -19,6 +19,35 @@ class Shift < ApplicationRecord
   before_destroy :notify_volunteers_about_destroy, prepend: true
   before_update  :notify_volunteers_about_update, prepend: true
 
+  def self.filtered_for_ngo(ngo, filters)
+    the_scope, order_by = filters
+    # unscoped is necessary because default_scope (deprecated?) is
+    # messing up the order by
+    shifts =
+      unscoped
```

Comment:
`unscoped` will include deleted shifts if not explicitly excluding them with e.g. `deleted_at not null`from what I remember.

We're using `paranoia` for soft deleting basically everything (insurance reasons) and therefore have a `default_scope` to neither include nor index marked as deleted records.